book/P1C4_survival.qmd

```diff
@@ -10,304 +10,486 @@ abstract: TODO (150-200 WORDS)
+
+
+| group  |id| $t^L$| $t$| $\delta$|mother |
+|-------:|-:|-----:|----:|-----:|:------|
+|       1|1|    55|  365|     0|dead   |
+|       1|2|    55|  365|     0|alive  |
+|       1|3|    55|  365|     0|alive  |
+|       2|4|    13|   76|     1|dead   |
+|       2|5|    13|  365|     0|alive  |
+|       2|6|    13|  365|     0|alive  |
+|       4|7|     2|   16|     1|dead   |
+|       4|8|     2|  365|     0|alive  |
+|       4|9|     2|  365|     0|alive  |
+
+: Excerpt of the `infants` [@pkgeha] time-to-event dataset.
+Rows are individual observations (id), `group` indicates matched infants, $t^L$ is the left-truncation time (time of inclusion into the study), $t$ is the observed time, $\delta$ is the event indicator. {#tbl-surv-data-infants}
+
 
-Left-truncation often occurs when study participation is conditional on the occurence of another event.
-This is best explained through examples:
+Now recall the calculation of the Kaplan-Meier estimate from @sec-surv-km.
+In @tbl-surv-data-infants, without stratifying according to mother's status, the first observed event time is $t_{(1)} = t_7=16$. Then ignoring left-truncation,
 
-In a study  from the 18th century @brostrom.influence.1987, when childhood and maternal mortality were relatively high, data was collected on infant survival during the first year after birth based on whether the mother was alive.
-Data was collected by adding an infant to the study when their mother died, then two other infants, whose mothers were alive, were matched into the study based on their age and features.
-The age of the infant at entrance into the study therefore signifies a *left-truncation event*, as infants who die before their mothers never enter the data and the *left-truncation time* is given by the infants' age at which the mother died.
+* $\calR_{t_{(1)}} = \{1,2,3,4,5,6,7,8,9\}$
+* $d_{t_{(1)}} = 1$
+* $n_{t_{(1)}} = 9$
+* $S(t_{(1)}) = 1 - \frac{1}{9} \approx 0.9$
 
-More formally, let $T^l_i$ the subject-specific left truncation time.
-Then we only observe subjects with $Y_i > T^l_i$ or $C_i > T^l_i$.
-Subjects with $Y_i < T^l_i$ never enter the data.
 
-Notably, left-truncation also plays an important role when modeling recurrent events or multi-state data, as the data generating process induces left-truncation (@sec-eha).
+In contrast, when we take left-truncation into account, subjects only enter the risk set for the event after their left-truncation time (we already know they survived until $t_i^L$ so they are not at risk for the event before that time), thus
 
+* $\calR_{t_{(1)}} = \{4, 5, 6, 7, 8, 9\}$
+* $d_{t_{(1)}} = 1$
+* $n_{t_{(1)}} = 6$
+* $S(t_{(1)}) = 1 - \frac{1}{6} \approx 0.8$
 
-#### Right-truncation {.unnumbered .unlisted}
+Thus, the definition of the KM estimator in @eq-km can still be used in case of left-truncation using the more general risk set definition $\calR_\tau = \{i: t_i^L \ \geq \tau \wedge t_i \geq \tau\}$
```

Comment:
```suggestion
Thus, the definition of the KM estimator in @eq-km can still be used in case of left-truncation using the more general risk set definition

$$
\calR_\tau = \{i: t_i^L \ \geq \tau \wedge t_i \geq \tau\}
$$ {#eq-risk-lt}
```